pisek/task_jobs/program.py

```diff
@@ -179,9 +182,9 @@ def _run_programs(self) -> list[RunResult]:
             os.close(fd)
             meta_files.append(meta_file)
 
-            running_pool.append(
-                subprocess.Popen(**pool_item.to_popen(minibox, meta_file))
-            )
+            popen = pool_item.to_popen(minibox, meta_file)
+            logger.debug("./" + " ".join(popen["args"]))
```

Comment:
This should probably contain some context, for example:

```suggestion
            logger.debug("Executing ./" + " ".join(popen["args"]))
```